modules/reference/pages/rpk/rpk-transform/rpk-transform-resume.adoc

```diff
@@ -0,0 +1,32 @@
+= rpk transform resume
+:description: rpk transform resume
+
+Resume a data transform.
+
+This command resumes execution of the specified data transform, if it was previously paused. Transform processors are restarted and resume processing from the last committed offset on the corresponding input partition.
+
+Subsequent `rpk transform list` operations will show transform processors as `running`.
+
+== Usage
+
+[,bash]
+----
+rpk transform resume [NAME] [flags]
+----
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|-h, --help |- |Help for resume.
+
+|--config |string |Redpanda or `rpk` config file; default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```

Comment:
```suggestion
|--config |string |Redpanda or `rpk` configuration file. Default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```

Reply:
Rejecting this change for consistency, as we would need to apply it globally. If necessary we can do it in another PR.